### PR TITLE
fix: add missing allowed category

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**

That was checked with the Prod Owner and this category should be allowed.

### CODE REVIEW

**Review by gallettilance — APPROVED** [review]

File: pkg/validation/internal/operatorhub.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ import (
 // * "Security"
 // * "Storage"
 // * "Streaming & Messaging"
+// * "Modernization & Migration"
```
Comment (tlwu2013) [inline pkg/validation/internal/operatorhub.go:46]: nit: should we keep this category list in alphabetical order?

File: pkg/validation/internal/operatorhub.go [hunk 2]
```diff
@@ -115,6 +116,7 @@ var validCategories = map[string]struct{}{
 	"Security":               {},
 	"Storage":                {},
 	"Streaming & Messaging":  {},
+	"Modernization & Migration": {},
```
Comment (tlwu2013) [inline pkg/validation/internal/operatorhub.go:119]: nit: should we keep this category list in alphabetical order?

Comment (exdx) [inline pkg/validation/internal/operatorhub.go:119]: nit: go fmt?

**Comment by camilamacedo86** [comment]
Hi @exdx, 

All done. Could you help us get this one merged?

**Review by dinhxuanvu — APPROVED** [review]
/approve

**Pull request merged**